anta/tests/interfaces.py

```diff
@@ -1342,3 +1342,81 @@ def _verify_interface_counters(self, interface_counters: dict[str, Any], interfa
                 self.result.is_failure(
                     f"{interface_failure_message_summary} - {counter['counter_name']} above threshold - Expected: {expected_counter_value} Actual: {counter_value}"
                 )
+
+
+class VerifyInterfacesTransceiverTemperature(AntaTest):
```

Comment:
Sorry for the back and forth but let's change the name of the test and follow the same structure as `VerifyInterfacesOpticsReceivePower`.
```suggestion
class VerifyInterfacesOpticsTemperature(AntaTest):
```